src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActionsSaveButton.tsx

```diff
@@ -1,54 +1,18 @@
-import { ContextModule } from "@artsy/cohesion"
 import { createFragmentContainer, graphql } from "react-relay"
-import { useSaveArtwork } from "Components/Artwork/SaveButton/useSaveArtwork"
 import { ArtworkActionsSaveButton_artwork$data } from "__generated__/ArtworkActionsSaveButton_artwork.graphql"
-import { ProgressiveOnboardingSaveArtwork } from "Components/ProgressiveOnboarding/ProgressiveOnboardingSaveArtwork"
-import { SaveUtilButton } from "Apps/Artwork/Components/ArtworkImageBrowser/SaveUtilButton"
-import { ArtworkActionsWatchLotButtonFragmentContainer } from "Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActionsWatchLotButton"
 import { ArtworkActionsSaveButtonV2FragmentContainer } from "Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActionsSaveButtonV2"
 import { ManageArtworkForSavesProvider } from "Components/Artwork/ManageArtworkForSaves"
-import { useCheckIfArtworkListsEnabled } from "Apps/CollectorProfile/Routes/Saves2/useCheckIfArtworkListsEnabled"
 
 interface ArtworkActionsSaveButtonProps {
   artwork: ArtworkActionsSaveButton_artwork$data
 }
 const ArtworkActionsSaveButton: React.FC<ArtworkActionsSaveButtonProps> = ({
   artwork,
 }) => {
-  const isArtworksListEnabled = useCheckIfArtworkListsEnabled()
-  const { handleSave } = useSaveArtwork({
-    isSaved: !!artwork.isSaved,
-    artwork,
-    contextModule: ContextModule.artworkImage,
-  })
-
-  const { isAuction, isClosed } = artwork.sale ?? {}
-  const isOpenOrUpcomingSale = isAuction && !isClosed
-  const isSaved = !!artwork.isSaved
-
-  if (isArtworksListEnabled) {
-    return (
-      <ManageArtworkForSavesProvider>
-        <ArtworkActionsSaveButtonV2FragmentContainer artwork={artwork} />
-      </ManageArtworkForSavesProvider>
-    )
-  }
-
-  // If an Auction, use Bell (for notifications); if a standard artwork use Heart
-  if (isOpenOrUpcomingSale) {
-    return (
-      <ArtworkActionsWatchLotButtonFragmentContainer
-        isSaved={isSaved}
-        artwork={artwork}
-        onClick={handleSave}
-      />
-    )
-  }
-
   return (
-    <ProgressiveOnboardingSaveArtwork>
-      <SaveUtilButton isSaved={isSaved} onClick={handleSave} />
-    </ProgressiveOnboardingSaveArtwork>
+    <ManageArtworkForSavesProvider>
+      <ArtworkActionsSaveButtonV2FragmentContainer artwork={artwork} />
```

Comment:
Over in `ArtworkActionsSaveButtonV2` we do have the [watch lot](https://github.com/artsy/force/blob/8953c9f1dde3bdbdb14cf47bbeeb52f51742e9bb/src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActionsSaveButtonV2.tsx#L63) and [progressive onboarding](https://github.com/artsy/force/blob/8953c9f1dde3bdbdb14cf47bbeeb52f51742e9bb/src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActionsSaveButtonV2.tsx#L73) code that is otherwise unreachable here, so the net effect should be the same even after all this deletion.